spark/v3.2/spark/src/main/java/org/apache/iceberg/spark/actions/BaseDeleteOrphanFilesSparkAction.java

```diff
@@ -107,10 +110,12 @@ public void accept(String file) {
   public BaseDeleteOrphanFilesSparkAction(SparkSession spark, Table table) {
     super(spark);
 
-    this.hadoopConf = new SerializableConfiguration(spark.sessionState().newHadoopConf());
-    this.partitionDiscoveryParallelism = spark.sessionState().conf().parallelPartitionDiscoveryParallelism();
+    hadoopConf = new SerializableConfiguration(spark.sessionState().newHadoopConf());
+    partitionDiscoveryParallelism = spark.sessionState().conf().parallelPartitionDiscoveryParallelism();
     this.table = table;
-    this.location = table.location();
+    includeHiddenPaths = PropertyUtil.propertyAsBoolean(table.properties(), INCLUDE_HIDDEN_PATHS,
+        INCLUDE_HIDDEN_PATHS_DEFAULT);
```

Comment:
I think it is confusing to include paths that are hidden, which is by default false. Instead I think it makes sense to disable ignoring hidden paths. That's why I suggested "ignore-hidden-paths" for the property above.